src/frontend/dune

```diff
@@ -1,47 +1,54 @@
 (library
-  (name frontend)
-  (public_name stanc.frontend)
-  (libraries core_kernel re menhirLib fmt middle analysis_and_optimization)
-  (inline_tests)
-  (preprocess (pps ppx_jane ppx_deriving.fold ppx_deriving.map)))
+ (name frontend)
+ (public_name stanc.frontend)
+ (libraries core_kernel re menhirLib fmt middle common
+   analysis_and_optimization)
+ (inline_tests)
+ (preprocess
+  (pps ppx_jane ppx_deriving.fold ppx_deriving.map)))
 
 (ocamllex lexer)
 
 (rule
  (targets parsing_errors.ml)
  (deps parser.mly parser.messages)
  (action
-  (with-stdout-to %{targets}
-   (run menhir
-    --explain
-    --strict
-    --unused-tokens parser.mly --compile-errors parser.messages))))
+  (with-stdout-to
+   %{targets}
+   (run menhir --explain --strict --unused-tokens parser.mly --compile-errors
+     parser.messages))))
 
 (menhir
  (modules parser)
  (flags :standard --table --strict --unused-tokens --fixed-exception))
 
 (rule
- (with-stdout-to parser_updated.messages
+ (with-stdout-to
+  parser_updated.messages
```

Comment:
What's the reason for these spacing changes? `run menhir` above collects the options to the same line but here the line is split into two.

Reply:
I think I ran ocamlformat on this file by accident, will revert